tests/acceptance/helpers/userSettings.js

```diff
@@ -176,6 +176,14 @@ module.exports = {
       return userId
     }
   },
+
+  getUsernameFromDisplayname: function(username) {
+    for (const userid in this.createdUsers) {
+      if (this.createdUsers[userid].displayname === username) {
+        return userid
+      }
+    }
+  },
```

Comment:
```suggestion
  getUsernameFromDisplayname: function(displayname) {
    for (const userid in this.createdUsers) {
      if (this.createdUsers[userid].displayname === displayname) {
        return userid
      }
    }
  },
```

Naming is hard. The function is called `getUsernameFromDisplayname` but it takes a parameter `username` and at the end does `return userid` - actually the logic is fine, the names just tricked me for a moment.